.circleci/config.yml

```diff
@@ -2,11 +2,8 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: docker.io/openwrtorg/packages-cci:v1.0.5
+      - image: docker.io/openwrtorg/sdk:ath79-generic
```

Comment:
Maybe we should have master somewhere in the version, like "sdk:master-ath79-generic" ?

Reply:
Added, the current layout is actually ath79-generic-master